packages/extension-browser/src/_locales/en/messages.json

```diff
@@ -186,5 +186,9 @@
     "targetUrl": {
         "message": "URL: $1",
         "description": "Label for the result page with the url analyzed."
+    },
+    "feedback": {
+        "message": "Feedback",
+        "description": "Label for the feedback button"
```

Comment:
Nit: I know the list isn't perfectly alphabetized, but if we can keep this roughly in alphabetical order I think that will help us find strings more easily in the future.